src/lib/database/store.ts

```diff
@@ -1039,7 +1043,7 @@ const initializer = immer<Database>((set, get) => ({
       ext_sdk_installation_id: params.ext_sdk_installation_id,
       phone_sdk_installation_id: installation_id,
       workspace_id: params.workspace_id,
-      user_identity_id: client_session.user_identity_id,
+      user_identity_id: client_session.user_identity_ids?.[0] ?? "",
```

Comment:
Will the real api return an empty string here if there is no matching user identity?

Reply:
No this won't return an empty string since it'll error from the check above. Will make it more explicit

Reply:
added explicit non null eslint ignore per line where the undefined case is thrown earlier

https://github.com/seamapi/fake-seam-connect/commit/61f50040f9c22f808c9f32f403378c5bb399bd71